content/news/2024-05-14-everse-workshop/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "EVERSE Satellite meeting (International Research Software Funders Workshop)"
+draft: false
+date: 2024-05-14T00:00:00Z
+start_date: 2024-05-14T00:00:00Z
+end_date: 2024-05-14T00:00:00Z
+summary: >
+    The main goal of the workshop is to serve as a platform for EVERSE to showcase the work of the project, and receive input from all relevant stakeholders, while facilitating networking and collaboration opportunities between all participants. Stakeholders of particular relevance in the workshop are Funders and Policy Makers, in order to increase awareness on the challenges and opportunities around research, as well as relevant community projects. The expectation is for participants to bring with them their communities’ best practices and services on research software (including but limited tos quality, management, sustainability, excellence, etc) in order to engage with EVERSE, while also establishing a bidirectional channel of communication moving forward.
```

Comment:
I would also add the punchier opening here, as this snippet appears on the front page - it should be the thing that grabs people's attention and makes them want to read more.

e.g.,

We are thrilled to announce our upcoming **EVERSE** workshop, to be held as a satellite event to the **International Research Software Funders Workshop** , and hosted by **SciLifeLab**, in **Uppsala, Sweden.** As EVERSE, we will showcase our project that will improve software quality for European sciences. We welcome input from Funders and Policy Makers as well as community projects. We want to hear about communities' best practices in research software, on managements, sustainability, excellence and more!